Cargo.toml

```diff
@@ -24,6 +24,7 @@ thiserror = "1.0"
 tree-sitter = "0.20"
 serde = "1.0"
 serde_json = "1.0"
+ansi_term = "0.12"
```

Comment:
I was going to say something glib about terminal colourization being _so important_, but it actually kind of is, as an accessibility thing. We highlight very little, but breaking it up into different colours makes it much easier to immediately bucket them appropriately in your head and focus your attention where you need it next.